### PR TITLE
[TS-167] added comments for GoDoc

### PR DESCRIPTION
**Pull request by jenchik**

### CODE REVIEW

**Review by pcherednichenko — DISMISSED** [review]

File: workers/workers.go [hunk 1]
```diff
@@ -8,25 +8,35 @@ import (
...
 
 type (
+	// Options structure for creation new worker.
 	Options struct {
 		Name     string
 		Schedule Schedule
 		Handler  func()
 	}
 
+	// The Schedule describes a job's duty cycle.
```
Comment (im-kulikov) [inline workers/workers.go]: `Schedule describes a job's duty cycle` comment must start from `name`

File: workers/workers.go [hunk 2]
```diff
@@ -8,25 +8,35 @@ import (
...
+
+	// ErrWrongOptions is returned by New calls
+	// when parameter Options.Schedule is NIL or Options.Handler is NIL.
+	ErrWrongOptions = errors.New("wrong options")
 
 	cronWorker = cron.New()
 	poolWorker = newPool()
 )
 
+// New returns a error if cannot create new worker
```
Comment (im-kulikov) [inline workers/workers.go]: mb `New returns an error if cannot create new worker`

File: workers/workers.go [hunk 3]
```diff
@@ -35,24 +45,36 @@ func New(opts Options) (err error) {
...
 }
 
+// Every returns a crontab Schedule that activates once every duration.
+// Delays of less than a second are not supported (will round up to 1 second).
+// Any fields less than a Second are truncated.
 func Every(duration time.Duration) Schedule {
 	return cron.Every(duration)
 }
 
+// Starting all workers.
```
Comment (im-kulikov) [inline workers/workers.go]: `Start all workers.` (because must start from `name`)

File: workers/workers.go [hunk 4]
```diff
@@ -35,24 +45,36 @@ func New(opts Options) (err error) {
...
 func Start() {
 	poolWorker.stop.Store(false)
 	cronWorker.Start()
 }
 
+// Stopping all workers.
```
Comment (im-kulikov) [inline workers/workers.go]: `Stop all workers`  (because must start from name)

File: workers/workers.go [hunk 5]
```diff
@@ -8,25 +8,35 @@ import (
...
 type (
+	// Options structure for creation new worker.
 	Options struct {
 		Name     string
 		Schedule Schedule
 		Handler  func()
 	}
 
+	// The Schedule describes a job's duty cycle.
 	Schedule interface {
```
Comment (im-kulikov) [inline workers/workers.go]: `Schedule = cron.Schedule`
what you think?

File: workers/workers.go (hunk 3 above)
Comment (pcherednichenko) [reply workers/workers.go]: the main thing is that it starts the same way, so it can be `Starting`, because there is word `Start`

**Review by im-kulikov — APPROVED** [review]

**Review by pcherednichenko — APPROVED** [review]

**Review by kpeo — APPROVED** [review]

**Pull request merged**